packages/Toggle/src/Toggle.tsx

```diff
@@ -403,12 +444,13 @@ export default class Toggle extends PureComponent {
     }
   };
   render() {
-    const checkboxId = this.props.id || this.checkboxId;
+    const checkboxId = this.props.id
+      ? this.props.id.toString()
```

Comment:
Is the ID here not converted to a string when rendered in HTML? Could be wrong but I don't think this is necessary.

Reply:
I think you're right, but TS complained, so 🤷‍♂

Reply:
I'm not in love for altering the output JS for the sake of TS complaining. It would be good to know specifically why it thinks that it can't be a number.